### ISSUE TITLE
Moving this into the FineUploader GitHub organization

### ISSUE THREAD

**Issue by rnicholus** [FineUploader/bower-dist#4]
It seems there may be a desire for users to pull in Fine Uploader via bower. That said, would you be open to joining the FineUploader organization and maintaining this repo as the official FineUploader bower distribution build? I'm happy to discuss more details if desired.

Also please note that, as of version 5.9, Fine Uploader is MIT licensed and 100% free open source software.

**Comment by feryardiant** [comment]
Hi @rnicholus,

Glad to hear that & it would be awesome to join in the official org. I'm in 😁 

Thanks for the opportunity.

**Comment by rnicholus** [comment]
Cool, good to hear! Are you willing/able to make some updates/improvements to make generation of the bower dist more automatic after this is moved into FineUploader? Ideally, when a new tag is pushed to the master branch of fine-uploader, the bower repo in the FineUploader org will be updated with the dist bytes. Or, a build script can be manually run that handles all of this, at least.

**Comment by feryardiant** [comment]
Actually that's was planned since I publish this repo, but that's couldn't happen without official permission.

Pada 14 Jun 2016 03.00, Ray Nicholus <notifications@github.com> menulis:

Cool, good to hear! Are you willing/able to make some updates/improvements to make generation of the bower dist more automatic after this is moved into FineUploader? Ideally, when a new tag is pushed to the master branch of fine-uploader, the bower repo in the FineUploader org will be updated with the dist bytes. Or, a build script can be manually run that handles all of this, at least.

**Comment by rnicholus** [comment]
Awesome! Here's the process I've been following:

1. I invite you to be an owner of the FineUploader-transfer organization. You transfer your bower repo into this org. Please do change the name of the repo to something like "bower" or "bower-dist". Happy to discuss names further.
2. I transfer the repo onto FineUploader. Then I make you an admin on this new repo in FineUploader and give you write/member access to fine-uploader. 
3. Issues are opened up by you and me to cover desired updates to the repo.

The reason for step 1 is that GitHub org permissions are not particularly well designed. The only way you can transfer a repo directly into FineUploader is if I give you unrestricted owner access to the entire org. There are some private repos in FineUploader that contain keys, passwords, etc that I'd like to keep private, and I gotten around to cleaning those up.

**Comment by feryardiant** [comment]
Okay, let's do this.

**Comment by feryardiant** [comment]
Done

**Comment by rnicholus** [comment]
I've transferred the repo into FineUploader and sent you an invite to be a member of the org. You will have write access to fine-uploader & bower-dist initially. After you join, I'll make you an admin of bower-dist.

**Comment by rnicholus** [comment]
I see you've accepted the invite. You're now an admin of this repo. Feel free to start creating issues or working on any changes as you see fit. I'll create issues myself in the coming days as needed. The workflow I tend to like involves opening pull requests with proposed changes, followed by discussion + adjustments (if needed) and then a merge into the develop or master branch (whichever is appropriate) once the feature/change is ready to go.

I suspect you will need to make adjustments to fine-uploader's build/travis configuration in order to automate all of this, but there may be other/better options. I'd love to discuss ideas you have - we can do so in the issue tracker.

**Comment by feryardiant** [comment]
Thanks @rnicholus.

I'll update the build & `README.md` late tonight. (GMT+7) 😉

**Comment by rnicholus** [comment]
Excellent. Good first step!

**Comment by feryardiant** [comment]
I've change my mind, just update everything. 😁
